libglusterfs/src/iobuf.c

```diff
@@ -570,6 +614,16 @@ iobuf_get_page_aligned(struct iobuf_pool *iobuf_pool, size_t page_size,
         req_size = iobuf_pool->default_page_size;
     }
 
+    if (req_size <= USE_IOBUF_POOL_IF_SIZE_GREATER_THAN) {
```

Comment:
You should check the result of `req_size + align_size`.

Reply:
will update in next patch

Reply:
done